### PR TITLE
Update releasing.md

### PR DESCRIPTION
**Pull request by uniyalmani**
## Summary

Update CI/CD from Travis CI to GitHub Actions in documentation.

## Details

- Updated Travis CI to GitHub Actions in documentation.

## Related Issue

Addresses #500

### CODE REVIEW

**Comment by aorinevo** [comment]
Hi @uniyalmani, much appreciate the contribution.  I did a quick search of 'travis' in the codebase and found a couple of more [instances](https://github.com/search?q=repo%3ANerdWalletOSS%2Fshepherd%20travis&type=code).  Can you take a look into them?

**Review by aorinevo — COMMENTED** [review]
See comments.

**Comment by aorinevo** [comment]
Also, just an FYI the CD portion is currently borked on account of permissions issues.  While we are able to land changes and tag them, we are not currently able to publish the tags to the npm registry. I'm working with folks at NerdWallet to get that resolved.

**Comment by uniyalmani** [comment]
> Hi @uniyalmani, much appreciate the contribution. I did a quick search of 'travis' in the codebase and found a couple of more [instances](https://github.com/search?q=repo%3ANerdWalletOSS%2Fshepherd%20travis&type=code). Can you take a look into them?

okay sir

**Comment by uniyalmani** [comment]
Updated remaining files as per your guidance.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <img alt="Illustration of a sheep" width=320 align=right src="https://user-images.githubusercontent.com/332258/46430732-c7bf5c80-c6fe-11e8-9cc8-2abc4e68dce2.jpg">
 
-![Travis status](https://img.shields.io/travis/NerdWalletOSS/shepherd/master.svg?style=flat-square)
+[![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/NerdWalletOSS/shepherd/release.yml?style=flat-square)](https://github.com/NerdWalletOSS/shepherd/actions)
```
Comment (aorinevo) [inline README.md:5]: cool!

**Review by aorinevo — APPROVED** [review]
LGTM!  Thank you, @uniyalmani!

Note, I will cherry-pick these changes into a branch on upstream and open a PR to main from there.  Once reviewed, approved, and merged, this PR will be auto-closed.

**Comment by aorinevo** [comment]
@uniyalmani, I need these commits to be squashed into 1.  Can you do me a solid and squash them?

**Comment by uniyalmani** [comment]
> @uniyalmani, I need these commits to be squashed into 1. Can you do me a solid and squash them?

Done

**Comment by aorinevo** [comment]
Looks like there are linting errors that need to be resolved. Can you take a look and resolve?

https://github.com/NerdWalletOSS/shepherd/actions/runs/7478742590/job/20354355273?pr=636

**Comment by aorinevo** [comment]
Closed by https://github.com/NerdWalletOSS/shepherd/pull/637

**Pull request closed without merging**